arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1660 @@
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/api-version"
+        }],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SharedAccessSignatureAuthorizationRule"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/exportDevices": {
+      "post": {
+        "tags": ["POST"],
+        "summary": "Export all the devices in an IotHub.",
+        "description": "Export all the devices in an IotHub.",
+        "operationId": "IotHubResource_ExportDevices",
+        "consumes": ["application/json", "text/json"],
+        "produces": ["application/json", "text/json"],
+        "parameters": [{
+          "name": "resourceName",
+          "in": "path",
+          "description": "The name of the resource.",
+          "required": true,
+          "type": "string"
+        }, {
+          "name": "exportDevicesRequest",
+          "in": "body",
+          "description": "The export devices request.",
+          "required": true,
+          "schema": {
+            "$ref": "#/definitions/ExportDevicesRequest"
+          }
+        }, {
+          "$ref": "#/parameters/subscriptionId"
+        }, {
+          "name": "resourceGroupName",
+          "description": "The name of the resource group.",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/api-version"
+        }],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/JobResponse"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/importDevices": {
+      "post": {
+        "tags": ["POST"],
+        "summary": "Import all the devices in an IotHub.",
+        "description": "Import all the devices in an IotHub.",
+        "operationId": "IotHubResource_ImportDevices",
+        "consumes": ["application/json", "text/json"],
+        "produces": ["application/json", "text/json"],
+        "parameters": [{
+          "name": "resourceName",
+          "in": "path",
+          "description": "The name of the resource.",
+          "required": true,
+          "type": "string"
+        }, {
+          "name": "importDevicesRequest",
```

Comment:
importDevicesParameters would be a better name and the description should also reflect that.